src/app/test/db/mongo/pruner_test.py

```diff
@@ -36,6 +38,60 @@ def test_prune_something(self, pruner, collection_mock):
         pruner.run()
         assert collection_mock.objects.return_value.no_cache.return_value.delete.called
 
+    def test_in_progress_ttl_index(self):
+        timeout = config._DB_SPEC["items"]["ttl"]["items"]["in_progress"].get("default")
+        test_pruner = MongoPruner(tasks=[task], ttl=timeout)
+        assert test_pruner._ttl == 15
+
+    def test_created_to_canceled(self, pruner):
+        test = Request(
+            system="T",
+            system_version="T",
+            instance_name="T",
+            namespace="T",
+            command="T",
+            created_at=datetime.datetime(2020, 5, 17),
+            status="CREATED",
+        )
+        test.save()
+        req_id = test.id
+        pruner.cancel_in_progress()
```

Comment:
I don't feel super strongly about this, but if `cancel_in_progress` is better suited to being an internal function (`_cancel_outstanding_requests`) then the function that we primarily want to be testing is the `run` function.  `run` is the public interface by which `_cancel_outstanding_requests` would be called, so that's the behavior you want to verify.  Said another way, confirming `_cancel_outstanding_requests` does what it's supposed to doesn't help you if the thing that actually gets called (`run`) never calls it.

More succinctly, write your tests so that you call `run` and verify that you get the intended behavior.